lib/puppet/provider/elasticsearch_plugin/plugin.rb

```diff
@@ -86,19 +86,10 @@ def install2x
     commands
   end
 
-  def install_options
-    return @resource[:install_options].join(' ') if @resource[:install_options].is_a?(Array)
-    return @resource[:install_options]
-  end
-
   def create
     es_version
     commands = []
     commands << @resource[:proxy_args].split(' ') if @resource[:proxy_args]
-    commands << install_options if @resource[:install_options]
-    if install_options.nil? or not install_options.include? 'es.path.conf'
-      commands << "-Des.path.conf=#{homedir}"
```

Comment:
This particular java option needs to stay in order to make plugins like Shield install their configurations into `/usr/share/elasticsearch` rather than `/etc/elasticsearch`